File: PHP/Creational/Abstract Factory/AppleProducts/CPU/LaptopCPU.php
<?php
/**
 * Created by PhpStorm.
 * User: argshub
 * Date: 3/20/2017
 * Time: 9:09 PM
 */

namespace AppleProducts\CPU;


class LaptopCPU implements CPU
{
    function getCPU() {
        return "Laptop CPU";
    }
}

File: JAVA/src/Creational/AbstractFactory/Products/ProductsCPU/LaptopCPU.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Creational.AbstractFactory.Products.ProductsCPU;

/**
 *
 * @author argshub
 */
public class LaptopCPU implements CPU {
    
    
    @Override
    public void getCPU() {
        System.out.println("Laptop CPU");
    }
}
